test/FunctionalTests/Tests/DataOData/Tests/OData.Writer.Tests/Atom/WriterMetadataDocumentAtomTests.cs

```diff
@@ -23,8 +23,9 @@ public class WriterMetadataDocumentAtomTests : ODataWriterTestCase
         [InjectDependency]
         public MetadataWriterTestDescriptor.Settings Settings { get; set; }
 
+        // For comment out test cases, see github: https://github.com/OData/odata.net/issues/883
         [Ignore] // Remove Atom

```

Comment:
If I understand correctly, this case was propagated from 6.x branch, not applicable to 7.x due to Atom, is ignored, so we are commenting out the TestMethod. Assuming Atom will never come back to 7.x, why we are not deleting this file (or even the Atom folder) completely, since we have it on 6.x already. Am I missing anything?